moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useAdvancedSearch.js

```diff
@@ -34,6 +34,13 @@ const makeAdvancedSearchWhereFilters = (filters) =>
 
       // Use operator name to get the GraphQL operator config for that operator
       const operatorConfig = FILTERS_COMMON_OPERATORS[operator];
+
+      if (!operatorConfig) {
+        throwFallbackComponentError(
+          `Operator ${operator} in this url does not exist.`
```

Comment:
could you update this error message to say "in this url no longer exists in the project list." to match up with the message for outdated fields? And also capitalize URL? that is my bad on first time around on the error messaging.

It is good to know that the field error handling and the operator error handling live so close together in the code. That should make a fancier feature like fixing the url and giving feedback about the fix easier in the future if needed.

Reply:
super happy to update it to that --  i remember thinking the wording for outdated fields was odd bc what if you just spelled it wrong, thats not an operator that used to exist but now doesnt, it just never existed :shrug: but maybe im overthinking and most ppl wouldnt get to this error by a misspelling in the url and its just a problem w ppl going to the url of old existing operators?

Reply:
that makes sense! i could have made it clearer that, in real life, this error would be encountered by a bookmarked URL or saved view (when that feature ships) that has old fields or operators that we no longer support.